.circleci/config.yml

```diff
@@ -2,42 +2,80 @@
 
 version: 2.1
 
-steps: &steps
+.install_bundler: &install_bundler
+  run:
+    name: Install a specific bundler version
+    command: gem install bundler -v 1.16.6
```

Comment:
This is fine but I couldn't remember the reason for this. Are we using a specific version rather than the latest so its predictable and ensure the same one is used for all builds?

Reply:
Yeah. The thing is, our base docker images are updated almost inmediately after bundler releases new versions. So, for example, we had previously `1.16.5` in the lock file at the `BUNDLED_WITH` section, but the images had already 1.16.6 installed. In that situation, the bundle binstub provided by the docker image was refusing to run with an error about the versions not matching. So yeah, in order to run a predictable version for all builds, and be able to update it at our own pace, I added this step installing the specific version that we want.

Reply:
Great thanks! I figured it was rather important/detailed. Do you think there would be value in having a comment in this file as to why we need a specific version? I ask because I'm so used to seeing that the latest is installed I figured its good to have a reason here. If not its fine.

Reply:
Sure! I'll improve the description of the step, the job is going to fail if we remove it I think, but a better description is never a bad idea :smiley:

Reply:
I'm actually not super happy about this because updating the bundler version we're using means updating it in all lock files, and here. Maybe a better idea is to lock the base docker images that we use, so they don't get updated unless we do it explicitly? :thinking:.